config.example.toml

```diff
@@ -425,6 +425,9 @@
 # a specific version.
 #ccache = false
 
+# List of tests or directories to exclude from the test suite. For example, exclude = ["tests/ui", "tests/debuginfo"];
+#exclude = []
```

Comment:
This doesn't seem accurate. From what I see you populated `config.skip` from this field, which also causes certain things to be skipped during the build not just tests.